recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -0,0 +1,37 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+do_fetch[depends] += "domd-agl-image-weston:do_${BB_DEFAULT_TASK}"
+
+################################################################################
+# Generic ARMv8
+################################################################################
+SRC_URI = "repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_ces2018/dom0.xml;scmdata=keep"
+
+###############################################################################
+# extra layers and files to be put after Yocto's do_unpack into inner builder
+###############################################################################
+# these will be populated into the inner build system on do_unpack_xt_extras
+XT_QUIRK_UNPACK_SRC_URI += "file://meta-xt-prod-extra;subdir=repo"
+# these layers will be added to bblayers.conf on do_configure
+XT_QUIRK_BB_ADD_LAYER += "meta-xt-prod-extra"
+
+XT_BB_LAYERS_FILE = "meta-xt-prod-extra/doc/bblayers.conf.dom0-image-minimal-initramfs"
+XT_BB_LOCAL_CONF_FILE = "meta-xt-prod-extra/doc/local.conf.dom0-image-minimal-initramfs"
+
+XT_BB_IMAGE_TARGET = "core-image-thin-initramfs"
+
+add_to_local_conf() {
```

Comment:
It is worth to comment, why are you adding separate function.
If I get it right, this is because `do_configure_append` is written in python, isn't it?

Reply:
yes, you got it right.
don't think this needs to be commented, it is seen from the function definitions